packages/cli/lib/project.ts

```diff
@@ -81,6 +81,7 @@ class ProjectIterator extends stream.Readable {
 
         return Promise.all(loadMetadata)
       })
+      .catch(e => this.emit('error', e))
```

Comment:
I did this because there were warnings because of unhandled rejections when API calls failed. This at least propagates the error through the stream

Reply:
Ah good. Is this why the pipeline would not fail when the projects fails to be fetched?

Reply:
Maybe? I did not really dig past that warning